justfile

```diff
@@ -78,5 +78,9 @@ build-docker-l1-geth:
 
 build-docker: build-docker-l1-geth build-docker-zkevm-node build-docker-zkevm-adaptor
 
+deploy-on-sequencer:
```

Comment:
@nomaxg This is the same as the `just deploy-rollup --force-recreate --renew-anon-volumes`, on line 17.

Reply:
done

Reply:
@nomaxg `deploy-rollup` already calls the `deploy-contracts` just command.

```
deploy-rollup *args: deploy-contracts
docker compose --profile zkevm1 --profile zkevm1-preconfirmations --env-file .env --env-file deployment.env up {{args}}
```

It's easy to overlook, after the colon.

Reply:
I see! So I guess this command isn't needed at all?